### PR TITLE
Update the information in contributing to the project

### PR DESCRIPTION
**Pull request by singh264**
Issue: https://github.com/eclipse-openj9/openj9/issues/19616
Co-authored-by: Henry Zongaro <zongaro@ca.ibm.com>
Co-authored-by: Tobi Ajila <atobia@ca.ibm.com>
Signed-off-by: Amarpreet Singh <Amarpreet.A.Singh@ibm.com>

### CODE REVIEW

**Comment by singh264** [comment]
@tajila how can I request your review?

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ To get started, read our [Contribution Guide](CONTRIBUTING.md).
...
+If you think you want to contribute but you're not ready to sign the Eclipse Contributor Agreement, why not come along to our weekly updates in the #planning channel, with calls at the time of new releases, to find out more about how we work. We talk about new ideas, answer any questions that get raised, and discuss project plans and status. We also do lightning talks on features and functions of the VM. Visit the *#planning* channel in our [Slack workspace](https://openj9.slack.com/) for information about upcoming community calls and minutes from previous meetings (Join [here](https://join.slack.com/t/openj9/shared_invite/enQtNDU4MDI4Mjk0MTk2LWVhNTMzMGY1N2JkODQ1OWE0NTNmZjM4ZDcxOTBiMjk3NGFjM2U0ZDNhMmY0MDZlNzU0ZjAyNzQ1ODlmYjg3MjA)).
```
Comment (tajila) [inline README.md:87]: Minor change "If you think you want to contribute but you're not ready to sign the Eclipse Contributor Agreement, why not join our weekly updates in the #planning channel? We also have calls at the time of new releases where you can learn more about how we work."

**Comment by singh264** [comment]
@tajila how can I re-request your review?

**Comment by singh264** [comment]
@tajila @pshipton how can I re-request your review?

**Review by tajila — APPROVED** [review]

**Review by pshipton — APPROVED** [review]

**Pull request merged**